### PR TITLE
INTLY-3680 introduce OSD install pipeline

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

**Comment by tremes** [comment]
The pipeline is available at https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/OSD/job/osd-cluster-integreatly-install. I haven't had a chance to run it yet.

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                  jobTemplate: 'Integreatly_Bootstrap_and_Install_[OSD]',
+                  templateType: 'workflow',
+                  importTowerLogs: true,
+                  removeColor: false,
+                  verbose: true,
+                  extraVars: """---
+                    tower_environment: dev
+                    rhsso_namespace: openshift-sso
+                    cluster_name: ${clusterName}
+                    integreatly_osd_uninstall_branch: ${uninstallationGitBranch}
```
Comment (pawelpaszki) [inline jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml]: should it be `installationGitBranch` (both extra var and the value)?

Comment (tremes) [reply jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml]: ahh good catch. Thanks

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    router_shard: ${routerShard}
+                    openshift_token: ${openShiftToken}
+                    github_client_id: ${githubClientId}
+                    github_client_secret: ${githubClientSecret}
+                    rhsso_admin_pass: ${customerAdminPassword}
+                    threescale_s3_access_key: ${3scaleAwsAccessKey}
+                    threescale_s3_secret_key: ${3scaleAwsAccessKey}
```
Comment (pawelpaszki) [inline jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml]: I think the value here should be: `3scaleAwsSecretKey`

**Comment by pawelpaszki** [comment]
see my comments. I will approve after successful execution (or at least failure not caused by the pipeline)

File: jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml (hunk 2 above)
Comment (tremes) [reply jobs/openshift/cluster/integreatly/osd-cluster-integreatly-install.yaml]: Ohh yeah week is approaching its end... :) Thanks.

**Comment by tremes** [comment]
Pipeline parameters can't start with a number. Updated.

**Comment by trepel** [comment]
LGTM, but it would be good to see successful execution. Try to reach out to @psturc , he might be in need to install on OSD so this job might be used instead of doing it directly via tower.

**Comment by trepel** [comment]
@psturc cool, thank you. It seems this is good to go then. Merging.

**Pull request merged**